lib/pymedphys/tests/mosaiq/test_mimicked_db.py

```diff
@@ -0,0 +1,78 @@
+# Copyright (C) 2021 Cancer Care Associates
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from pymedphys._imports import numpy as np
+from pymedphys._imports import pytest
+
+from pymedphys._mosaiq import helpers
+
+from . import _connect
+from .data import mimics
+
+PATIENT_ID = 989898
+FIELD_ID = 88043
+A_TREATMENT_TIME = "2020-04-27 08:03:28.513"
+MACHINE_ID = "2619"
+
+
+@pytest.fixture(name="create_mimic_db_with_tables")
+def create_mimic_db_with_tables_base():
+    """ will create the test database, if it does not already exist on the instance """
+    mimics.create_db_with_tables()
+
+
+@pytest.mark.mosaiqdb
+def test_get_patient_name(
+    create_mimic_db_with_tables,
+):  # pylint: disable = unused-argument
+
+    with _connect.connect(database=mimics.DATABASE) as connection:
+        name = helpers.get_patient_name(connection, PATIENT_ID)
+        assert name == "PHYSICS, Mock"
```

Comment:
Hi @rembishj,

Some cool news, this PR is able to be used to test the `helpers` functions calling from a testing MSSQL database! 🙂 🥳

Reply:
Awesome! I'll probably need to meet with you to explain everything that's going on here 😄